docnado/docnado.py

```diff
@@ -944,6 +944,10 @@ def main():
                         default=False,
                         help='Specify a port for the docnado server')
 
+    parser.add_argument('--nonlocal', action="store_true", dest='non_local',
```

Comment:
What's your feelings on changing this to something akin to Jekyll

`--host={ip_address}`  maybe even `--host={ip_address}:{port}`

to give user's a bit more flexbility

Reply:
Sorted. Syntax is
`--host '0.0.0.0' --port 9000`

This maintains compatibility with older versions which only have `--port`.